docs/sources/tempo/_index.md

```diff
@@ -55,5 +56,3 @@ Tempo integrates well with a number of open source tools:
 ## Explore
 
 {{< card-grid key="cards" type="simple" >}}
-
-<p align="center"><img src="/static/img/search/tempo.svg"  alt="Tempo Logo"></p>
```

Comment:
No replacement for this one?

Reply:
I'll let @knylander-grafana and the team decide but to me it looks like a rogue image at the end of the page.
I don't see similar images in https://grafana.com/docs/grafana/latest/, https://grafana.com/docs/grafana/latest/alerting/, or https://grafana.com/docs/loki/latest/.

<img width="815" alt="Screenshot 2024-09-02 at 11 06 45" src="https://github.com/user-attachments/assets/9c597f1c-f0f0-4647-a119-73277a3a7eaf">

Reply:
This was the first landing page that I did and I just liked the addition of the Tempo logo. Bookends. :) We can remove it, if you like, since we don't use this kind of framing on any other landing pages.